### ISSUE TITLE
macos select dropdowns are native but not within a11y guidelines

### ISSUE THREAD

**Issue by caolanm** [CollaboraOnline/online#15987]
<img width="206" height="243" alt="Image" src="https://github.com/user-attachments/assets/3a012572-2cc4-413c-9d81-b4c68672f419" />

technically doesn't have sufficient contrast between the text and background in a highlighted row for a11y contrast guidelines as the contrast ratio against the background falls below the required value of 4.5:1.

(as measured on macos with chrome using [Colour Contrast Analyser](https://github.com/ThePacielloGroup/CCAe))

Our own combobox dropdown *do* have the required contrast ratio. If the native select dropdown is restyled/replaced to render like our own combobox dropdown then they are consistent with that and look like:

<img width="1667" height="335" alt="Image" src="https://github.com/user-attachments/assets/562aefa5-c2d4-45fa-8d36-178fd9735163" />

<img width="171" height="288" alt="Image" src="https://github.com/user-attachments/assets/69291f1c-dac3-4eef-b23f-90d693cf23e9" />

<img width="86" height="110" alt="Image" src="https://github.com/user-attachments/assets/063c7820-aacb-4772-82aa-dcec1b6812a6" />
